corehq/ex-submodules/casexml/apps/case/tests/test_bugs.py

```diff
@@ -27,7 +28,6 @@ def testConflictingIds(self):
         """
         If two forms share an ID it's a conflict
         """
-        self.assertEqual(0, len(CommCareCase.view("case/by_user", reduce=False).all()))
```

Comment:
all of these tests were asserting that `delete_all_cases` called in setup actually deleted them all. seemed silly to do it everywhere and this should make tests run (very slightly) faster